src/lib/pages/schedule-view.svelte

```diff
@@ -115,8 +115,7 @@
     ];
 
   let coreUser = coreUserStore();
-  $: editDisabled =
```

Comment:
I think this is fine as is. `DT-3050` looks like it's referring to some logic in Cloud (not OSS). Left a comment on the task and I think we can close this PR.